.github/workflows/preview-apk.yml

```diff
@@ -0,0 +1,48 @@
+name: Upload Preview APK
+
+on: pull_request
+
+jobs:
+  build:
+    name: Upload Preview APK
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: textbook/git-checkout-submodule-action@master
+      - run: cp jni/deltachat-core-rust/rust-toolchain .
+      - uses: actions-rs/toolchain@v1
+      - uses: Swatinem/rust-cache@v1
+        with:
+          working-directory: jni/deltachat-core-rust
+      - uses: actions/setup-java@v1.4.3
+        with:
+          java-version: 11
+      - uses: android-actions/setup-android@v2
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - uses: nttld/setup-ndk@v1
+        id: setup-ndk
+        with:
+          ndk-version: r21d
+
+      - name: Compile core
+        env:
+          ANDROID_NDK_ROOT: ${{ steps.setup-ndk.outputs.ndk-path }}
+        run: |
+          export PATH="${PATH}:${ANDROID_NDK_ROOT}/toolchains/llvm/prebuilt/linux-x86_64/bin/:${ANDROID_NDK_ROOT}"
+          ./scripts/install-toolchains.sh && ./ndk-make.sh armeabi-v7a
```

Comment:
Why only one architecture? I think we should switch to simply `ndk-make.sh` before merging, so it generates universal APK.

Reply:
I'm one minute late, but anyway.

Reply:
yikes! :( well I was using `ndk-make.sh` without args, but since in the comments inside `ndk-make.sh` seemed that for testing it is common to only use one arch to speed up things used that, what we do now? do I open another PR or just commit directly to master?

Reply:
Just commiting to master is ok